src/locales/en.json

```diff
@@ -530,6 +534,13 @@
                 "address": "Invalid address. Reward address must be a valid P chain address.",
                 "validating": "Your registered node: {nodeID} is already validating.",
                 "suspended": "Your registered node: {nodeID} is suspended."
+            },
+            "pending_multisig": {
+                "signed": "Signed",
+                "pending": "Pending",
+                "threshold": "{value}  out of  {threshold}  co-owners in your Multisig must sign the transaction that your validator node can be succesfully registered.",
```

Comment:
{value} here should be replaced by {threshold}
{threshold} should be replaced by {total owners in multi-sig wallet}